graph.cpp

```diff
@@ -65,7 +99,7 @@ void Graph::update(GraphUpdate upd) {
 }
 
 Supernode* Graph::generate_delta_node(uint64_t node_n, long node_seed, uint64_t
-                                src, const std::vector<uint64_t>& edges) {
+							 src, const std::vector<uint64_t>& edges, Supernode *delta_loc) {
```

Comment:
if we're using placement construction, why are we returning anything?

Reply:
I suppose we don't need to. Do you think we should make this a void function?

Reply:
I'll just make it void